admin/src/store/helpers.js

```diff
@@ -0,0 +1,135 @@
+import Vue from 'vue';
+
+export const STATUSES = {
+    pending: "PENDING",
+    success: "SUCCESS",
+    error: "ERROR",
+};
+
+/**
+ * Helper function to log Axios error in dev tools
+ * @param {AxiosError} error
+ */
+function logError(error) {
```

Comment:
you already have the same functionality in `admin/src/api/connections.js `

Reply:
removed from `connections.js`